lang/en.json

```
@@ -436,6 +436,10 @@
         "MatomoConfigurationMatomoDisableCookiesDescription": "Disables all first party cookies.",
         "MatomoConfigurationMatomoRequireConsentTitle": "Require tracking consent",
         "MatomoConfigurationMatomoRequireConsentDescription": "Track only when user gave tracking consent. In a consent screen you need to call \"window._paq=window._paq||[];window._paq.push(['rememberConsentGiven']);\" when the user gives consent.",
+        "MatomoConfigurationMatomoCustomCookieTimeOutEnableTitle": "Enable custom visitor Cookie timeout",
```

@tomper00 Can we update the text case and add a fullstop at the end to make it consistent ?
```suggestion
        "MatomoConfigurationMatomoCustomCookieTimeOutEnableTitle": "Enable Custom Visitor Cookie Timeout.",
```